projects/tags-input-demo/src/app/app.component.scss

```diff
@@ -30,3 +30,42 @@
     }
   }
 }
+
+.users-list {
+  padding: 8px 0;
+  width: 100%;
+  box-shadow: grey;
+  background: white;
+  overflow: auto;
+  border-bottom: 0.5px solid grey;
+  &__item {
```

Comment:
Since you'll create a new component for this, these styles for `&__item` can go inside that component.